openedx/core/djangoapps/discussions/api/data.py

```diff
@@ -0,0 +1,23 @@
+import attr
+
+
+class ObjectDoesNotExist(Exception):
+    """
+    Imitating Django model conventions, we put a subclass of this in some of our
+    data classes to indicate when something is not found.
+    """
+    pass
```

Comment:
I currently don't see a use case for this. If a mapping doesn't exist we'll just return None.

Reply:
Removed

Reply:
Removed